javascript/javascript1/week2/formal_fullname.js

```diff
@@ -0,0 +1,13 @@
+const namethatIwant = getFullname('Benjamin', 'Hughes', true);
+
+function getFullname(firstName, lastName, formalFullName) {
+    if (formalFullName == true) {
```

Comment:
use `===` to compare the value and type.
You can also shortly write it `if (formalFullName) {`